srcpkgs/seatd/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'seatd'
+pkgname=seatd
+version=0.4.0
+revision=1
+build_style=meson
+configure_args="-Dexamples=disabled $(vopt_if elogind -Dlogind=enabled)"
```

Comment:
elogind support should be compiled in by default to support switch to build wlroots w/ libseat backend only; see #26224

Reply:
it should be enabled by default:
> build_options_default="elogind"